recipes/duc/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "duc" %}
+{% set version = "1.4.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://github.com/zevv/duc/releases/download/{{ version }}/duc-{{ version }}.tar.gz"
+  sha256: f4e7483dbeca4e26b003548f9f850b84ce8859bba90da89c55a7a147636ba922
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - autoconf
+    - automake
+    - make
+    - pkg-config
+  host:
+    - tokyocabinet
+    - ncurses
+    - cairo
+    - glib
+    - pango
+    - xorg-libx11
+  run:
+    - tokyocabinet
+    - ncurses
+    - cairo
+    - glib
+    - pango
+    - xorg-libx11
```

These should the be added automatically via run_exports.
```suggestion
    - pango
    - xorg-libx11
```